### PR TITLE
DM-33557:  Add cp_verify tests to ci_cpp

### PR DESCRIPTION
**Pull request by czwa**
* Add verify commands to most stages.  PTC dependent models are skipped for now, until a proper PTC curve is added to the testdata.
* Add target results files for unit test checking of calculated results, with instructions on how to update if needed.

### CODE REVIEW

File: tests/data/bfkDet.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - -8.768544056471406
+  - -10.341225414260292
+  - -8.858106359058388
+  - -8.93306446747915
+  - -8.777269141371395
+  - -12.154781876678193
+  - -11.286575171837608
+  NUM_MATCHES: 22
+  SIZE_DIFF:
+  - .nan
```
Comment (erykoff) [inline tests/data/bfkDet.yaml:46]: Why are there nans in here?  And how do those compare?

File: tests/data/linearityDet.yaml [hunk 2]
```diff
@@ -0,0 +1,981 @@
...
+  C00:
+    FIT_PARAMS:
+    - 25087.503340648258
+    FIT_PARAMS_ERR:
+    - 0.0
+    FIT_RESIDUALS:
+    - 0.00016276041696983157
+    - -0.00032552083393966313
+    - 0.00016276041606033687
+    - .nan
```
Comment (erykoff) [inline tests/data/linearityDet.yaml:11]: Also why are there nans here?

File: tests/data/ptcDet.yaml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    AMP_NOISE: 10.0
+    PTC_BFE_A00: -1.1833364688792843e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0936583586522444
+    PTC_NOISE: 15.381025016469493
+    PTC_TURNOFF: 118690.37014599473
+  C07:
+    AMP_GAIN: 1.1033
+    AMP_NOISE: 8.5
+    PTC_BFE_A00: .nan
```
Comment (erykoff) [inline tests/data/ptcDet.yaml:61]: This is also all nans in the linearity?

**Review by erykoff — APPROVED** [review]
Are all the nans in the comparison related to the instability ticket?  We need to figure out what to do about that.

File: tests/data/bfkDet.yaml (hunk 1 above)
Comment (czwa) [reply tests/data/bfkDet.yaml:46]: This is in the difference in size between the brighter-fatter corrected and uncorrected measurements.  A look at the associated magnitudes show that these are on the bright end, so it's likely a failure to measure the size in one (both?) of those (there's no check).  I also see that `cp_verify` is still using SDSS sizes, and I thought those had been deprecated in favor of the HSM sizes, so there's at least two issues to resolve there.  https://jira.lsstcorp.org/browse/DM-42524 will sort out these issues.

File: tests/data/linearityDet.yaml (hunk 2 above)
Comment (czwa) [reply tests/data/linearityDet.yaml:11]: These nans appear to be from the linearity code masking all of these values (the values are taken directly from the calibration).  I'd have to sit down and debug the linearity fits to figure out why.  https://jira.lsstcorp.org/browse/DM-42525

File: tests/data/ptcDet.yaml (hunk 3 above)
Comment (czwa) [reply tests/data/ptcDet.yaml:61]: This I'm not surprised by:  C07 has a bad cameraGeom values, so it generally gets masked.  I can't remember off hand if this is a saturation issue or a read noise issue, but they're both wrong and so this amplifier is bad in ci_cpp processing.  It should be unrelated to the bad linearity fits.

File: tests/data/bfkDet.yaml (hunk 1 above)
Comment (czwa) [reply tests/data/bfkDet.yaml:46]: I forgot to add: nan values in either the as-run or as-expected results are excluded from comparisons.  This is one of the workarounds for the stability issue.

**Pull request merged**